src/component/Display.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Display(props) {
+  const { result } = props;
+
+  return (
+    <div className="Display setHeight">
+      <p>{result}0</p>
```

Comment:
**Friendly reminder**
This is from the previous milestone, you are supposed to be using defaultProps and propTypes here.